app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -129,11 +131,18 @@ class BookmarksActivity : DuckDuckGoActivity() {
         val message =
             Html.fromHtml(getString(R.string.bookmarkDeleteConfirmMessage, bookmark.title))
         val title = getString(R.string.bookmarkDeleteConfirmTitle)
-        deleteDialog = alert(message, title) {
```

Comment:
this was using Anko @CDRussell, in case you had it in your task to get rid of it